ramp4-pcar4/ramp4-pcar4
# CSS Enhancements

### Related Item(s)
#2377, #2431 

### Changes
- Fixed an issue where the map caption would shrink without the attribution.
- Refactored demos CSS and put common styling in `.css` files.
- Removed unused styling.

### QA Testing
Please test against `main` branch https://ramp4-pcar4.github.io/ramp4-pcar4/main/demos/enhanced-samples.html

### Testing
For #2431:
1. Go to https://mohsin-r.github.io/ramp4-pcar4/css-tweaks/demos/index-samples.html?sample=7
2. Verify that there is no gap between the map caption and the appbar.

For #2377:
Open the various demos pages and verify that all styling looks ok.
